utils_test.py

```diff
@@ -188,3 +191,48 @@ def test_get_thumbnail_icon_url_for_category(self):
         self.assertEqual(
             utils.get_thumbnail_icon_url_for_category('Nonexistent'),
             '/images/subjects/Lightbulb.svg')
+
+    def test_get_current_date_as_string(self):
+        custom_datetimes = [
+            datetime.date(2011, 01, 01),
```

Comment:
(Had originally added extra zeros as an inspiration to test for multiple cases. But your comment made me realize that it might be futile to add zeros here, since what needs to be tested is that the format of string being returned is correct or not)
Removed